### PR TITLE
A small collection of minor changes

### PR DESCRIPTION
**Pull request by ximinez**

### CODE REVIEW

File: src/ripple/app/misc/impl/TxQ.cpp [hunk 1]
```diff
@@ -1116,12 +1093,12 @@ TxQ::processClosedLedger(Application& app,
         return;
     }
 
+    std::lock_guard<std::mutex> lock(mutex_);
```
Comment (scottschurr) [inline src/ripple/app/misc/impl/TxQ.cpp]: Here's the crux of the change, so `feeMetrics_.update()` happens under the lock?

File: src/ripple/app/misc/TxQ.h [hunk 2]
```diff
@@ -424,7 +420,8 @@ class TxQ
             TxQAccount::TxMap::iterator, std::uint64_t feeLevelPaid,
                 PreflightResult const& pfresult,
                     std::size_t const txExtraCount, ApplyFlags flags,
-                        beast::Journal j);
+                        beast::Journal j,
+                            std::lock_guard<std::mutex> const& lock);
```
Comment (ximinez) [inline src/ripple/app/misc/TxQ.h]: That's a good point. Will fix.

File: src/ripple/app/misc/impl/TxQ.cpp (hunk 1 above)
Comment (ximinez) [inline src/ripple/app/misc/impl/TxQ.cpp]: Yep.

**Comment by scottschurr** [comment]
I like the `snapshot`.  I have a non-critical naming question.  Still :+1:.

**Comment by nbougalis** [comment]
Merged as 3a973ab71911e2b0fdd31f58590d25aa03060e8b, 6f10fe8502afe8b29b0aba6199086d04a7b8e45f and d471e533b712bd539fe5070f92c7688a27749158.

**Pull request closed without merging**